trymito.io/components/Footer/Footer.tsx

```diff
@@ -37,11 +37,14 @@ const Footer = (): JSX.Element => {
                         Product
                     </ol>
                     <li className={classNames(footerStyle.nav_item)}>
-                        <Link href='/spreadsheet-automation'>Automation</Link>
+                        <Link href='/spreadsheet-automation'>Mito Spreadsheet</Link>
                     </li>
                     <li className={classNames(footerStyle.nav_item)}>
                         <Link href='/python-ai-tools'>Mito AI</Link>
                     </li>
+                    <li className={classNames(footerStyle.nav_item)}>
```

Comment:
Can we call this "Infrastructure Integration"? I think custom functionality points more towards a consulting thing.